src/20_Components/amp-o2-player.html

```diff
@@ -0,0 +1,39 @@
+<!--
+  #### Introduction
+
+  Embed o2 player in your AMP HTML files. You can find player here http://on.aol.com/,
```

Comment:
use markdown to format links:

" Embed the [o2 player](http://on.aol.com/) in your AMP HTML files. You can find more details in the documentation on [Github](https://github.com/ampproject/amphtml/blob/master/extensions/amp-o2-player/amp-o2-player.md)."